### DISCUSSION TITLE
如何安装Steam++？（电脑端）

### DISCUSSION THREAD

**Discussion by scrbyq** [BeyondDimension/SteamTools#917] [Ideas]
我不会在电脑上安装Steam++，求大佬帮忙安装最新版本以便拥有Steam更好的游戏体验！

**Comment by scrbyq** [comment]
求大佬帮忙！感激

**Comment by AigioL** [comment]
在官网或 Github/Gitee releases 上下载最新版本的 安装包或压缩包
压缩包解压到任意文件夹中就算安装完成了
安装包一直点下一步就行了

**Comment by elarbor** [comment]
这不是官网吗？[https://steampp.net/](https://steampp.net/)
![image](https://github.com/user-attachments/assets/7f1bcd50-5591-4941-9736-25698c498e64)
